webpack.config.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Decorates the default webpack configuration (coming from funcpack).
+ * See https://github.com/Azure/azure-functions-pack
+ */
+module.exports = function(config, webpack) {
+  // fix an error with npm modules that call GENTLY.hijack(require)
```

Comment:
if there's an open bug/issue somewhere pls link it in the code so in the future we can check if it's been fixed

Reply:
right. note that "casually" in the funcpack readme there's an example with the fix to this bug :)